modules/indexing/src/test/java/org/apache/ignite/testsuites/IgniteBinarySimpleNameMapperCacheQueryTestSuite2.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.testsuites;
+
+import org.apache.ignite.testframework.config.GridTestProperties;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * Cache query suite part 2 with binary marshaller.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({IgniteBinaryCacheQueryTestSuite2.class})
+public class IgniteBinarySimpleNameMapperCacheQueryTestSuite2 {
+    /** */
+    @BeforeClass
+    public static void init() {
+        GridTestProperties.setProperty(GridTestProperties.BINARY_MARSHALLER_USE_SIMPLE_NAME_MAPPER, "true");
```

Comment:
Do we really need this new suite?
Is it added on TC?
Do we need to clear property after execution?

Reply:
I run this suite locally and get many test failures. Can, you, please, double check it passes both locally and on TC?

Reply:
I believe that creating a suite with a simplemapper setting enabled for IgniteBinaryCacheQueryTestSuite2 was simply forgotten. As I understand, IgniteBinaryCacheQueryTestSuite was split into 2 parts due to time constraints on teamcity.
But, since there are errors when you turn it on, it makes sense to create a separate task for this. Now I removed it from patch.